src/ess/reduce/time_of_flight/workflow.py

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2025 Scipp contributors (https://github.com/scipp)
+from collections.abc import Iterable
+
+import sciline
+
+from ..nexus import GenericNeXusWorkflow
+from .eto_to_tof import default_parameters, providers
+
+
+def GenericTofWorkflow(
+    *,
+    run_types: Iterable[sciline.typing.Key] | None = None,
+    monitor_types: Iterable[sciline.typing.Key] | None = None,
+) -> sciline.Pipeline:
+    """ """
+    wf = GenericNeXusWorkflow()
```

Comment:
I think this may be slow unless you forward the `run_types` etc.? Not sure if you then need to prune again, or if there is a better way.

Reply:
I opted for pruning twice.